### ISSUE TITLE
11.1 libpq build without ssl support?

### ISSUE THREAD

**Issue by booleanbetrayal** [PostgresApp/PostgresApp#507]
OSX / 2.2.2 (51)

Getting this error from a connection string that worked on a previous, 10.x bundled version of Postgres.app:

`PG::ConnectionBad: sslmode value "verify-full" invalid when SSL support is not compiled in`

This seems to point to the possibility of `libpq` being configured without `--with-openssl`. Wondering if this is the case by mistake in these recent builds?

**Comment by booleanbetrayal** [comment]
Traced it to another dependency

**Comment by joelvh** [comment]
@booleanbetrayal what's the dependency you were able to track this down to and did you solve it? Ever since I switched from the Homebrew version to this, I am having this issue too.

**Comment by booleanbetrayal** [comment]
@joelvh - So the issue I was having from a Rails process that was connecting to a local PostgresApp installation. In the end, it actualy turned out to be the ruby version I was using had been recompiled, but without proper SSL support (built against OpenSSL). Rebuilding the Ruby binary allowed the Rails client app to connect correctly. So, very environmental, but hopefully that helps!

**Comment by joelvh** [comment]
OK, thanks @booleanbetrayal. What I discovered in my case was that when the `pg` gem was installed, it was installed with the Homebrew version of postgres. I uninstalled that and installed Postgres.app and started getting the error probably because the `pg` gem was now referencing a different version of the postgres drivers. Uninstalling and reinstalling the `pg` gem seems to have fixed it (presumably because it was not compiled using the Postgres.app drivers).
